community/unsafe/src/main/java/org/neo4j/unsafe/impl/internal/dragons/MemoryManager.java

```diff
@@ -32,8 +32,7 @@
     /**
      * The amount of memory, in bytes, to grab in each Slab.
      */
-    private static final long GRAB_SIZE = Integer.getInteger(
-            MemoryManager.class.getName() + ".GRAB_SIZE", 512 * 1024 ); // 512 KiB
+    private static final long GRAB_SIZE = FeatureToggles.getInteger( "GRAB_SIZE", 512 * 1024 ); // 512 KiB
```

Comment:
and as soon as you will rename your class IDE will not gonna be able to help you with that, do not see how documenting that behaviour will help to prevent people mistakes.